src/Symfony/Component/Config/Definition/Builder/EnumNodeDefinition.php

```diff
@@ -36,6 +36,17 @@ public function values(array $values): static
         return $this;
     }
 
+    public function enumClass(string $enumClass): static
+    {
+        if (!enum_exists($enumClass)) {
+            throw new \InvalidArgumentException(sprintf('The enum class "%s" does not exist.', $enumClass));
+        }
+
+        $this->values = [$enumClass];
```

Comment:
Why don't we introduce a dedicated property for the enum class? This honestly feels like a hack.